src/datagrid/column-service/column-service.jsx

```diff
@@ -240,6 +262,10 @@ export default {
       case 'multiselect': {
         const selectOptions = col.selectComponentOptions
           || selectComponentOptions.get(column.columnKey);
+        invariant(
```

Comment:
This is not always an error case for example when values are filtered based on outer coming selections/filters.

Reply:
Ok, didn't think of that.